ctapipe_io_lst/__init__.py

```diff
@@ -18,16 +18,26 @@
 )
 
 from ctapipe.io import EventSource
-
-from .containers import LSTDataContainer, PixelStatusContainer
-
+from ctapipe.core.traits import Int
+from .containers import LSTDataContainer
+from ctapipe.io.containers import PixelStatusContainer
 
 __all__ = ['LSTEventSource']
 
 
 class LSTEventSource(EventSource):
     """EventSource for LST r0 data."""
 
+    n_gains = Int(
+        2,
+        help='Number of gains at r0/r1 level'
+    ).tag(config=True)
+
+    baseline = Int(
+        400,
+        help='r0 waveform baseline '
```

Comment:
Hi Franca, the LST EvB2 data, to which no correction has been applied has a baseline of 300, right?

Reply:
Now the baseline is defined by Pawel's code and it is 300, but in the future this will be set by the EvB3 and Julien Houle said to me that he will use a baseline at 400 (hardcoded in the EvB), 
if he will change his mind we can always change this value (I said it is not used for the moment)